src/Util/Common.php

```diff
@@ -52,15 +52,57 @@ public static function isPharFile($path)
 
 
     /**
+     * Get absolute path, do not check exsit
+     *
+     * @param string $path The path to use.
+     *
+     * @return mixed
+     */
+    public static function absolutePath($path)
+    {
+        // Don't prefix absolute paths.
+        if (substr($path, 0, 1) !== '/') {
+            $path = getcwd().'/'.$path;
+        }
+
+        // Replace backslashes with forwardslashes.
+        $path = str_replace('\\', '/', $path);
+        // Combine multiple slashes into a single slash.
+        $path = preg_replace('/\/+/', '/', $path);
```

Comment:
you could canonicalize backslashes with this as well in one. Combining multiple slashes is much appreceated.